### ISSUE TITLE
Indonesian language

### ISSUE THREAD

**Issue by Tunous** [Tunous/SwipeNews-Issues#22]

**Comment by Chidaverz** [comment]
Hi, Can I work on this?

**Comment by Tunous** [comment]
Yes! Translations can be made using this link: https://www.localize.im/v/vb

**Comment by Tunous** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Use contributed translations](https://trello.com/c/3FaOporP/823-use-contributed-translations)
